hdijupyterutils/hdijupyterutils/configuration.py

```diff
@@ -20,49 +21,71 @@ def wrapped_f(*args):
                 return overrides[name]
             else:
                 return f(*args)
-        
+
         # Hack! We do this so that we can query the .__name__ of the function
         # later to get the name of the configuration dynamically, e.g. for unit tests
         wrapped_f.__name__ = f.__name__
         return wrapped_f
-    
+
     return ret
 
-    
+
 def override(overrides, path, config, value, fsrw_class=None):
     """Given a string representing a configuration and a value for that configuration,
     override the configuration. Initialize the overrided configuration beforehand."""
     _initialize(overrides, path, fsrw_class)
     overrides[config] = value
 
-    
+
 def override_all(overrides, new_overrides):
     """Given a dictionary representing the overrided defaults for this
     configuration, initialize the global configuration."""
     overrides.clear()
     overrides.update(new_overrides)
 
 
+def merge_required(overrides, path, fsrw_class=None):
```

Comment:
to be more explicit, can we make this `merge_required_session_configs`?

Reply:
yessir - good call